hack/make/build-deb

```diff
@@ -86,6 +88,11 @@ set -e
 		cat >> "$DEST/$version/Dockerfile.build" <<-EOF
 			RUN cp -aL hack/make/.build-deb debian
 			RUN { echo '$debSource (${debVersion}-0~${suite}) $suite; urgency=low'; echo; echo '  * Version: $VERSION'; echo; echo " -- $debMaintainer  $debDate"; } > debian/changelog && cat >&2 debian/changelog
+		EOF
+		if [[ ${new_systemd[@]} =~ $version ]]; then
+			echo "RUN sed -i -- 's/#TasksMax=infinity/TasksMax=infinity/' contrib\/init\/systemd\/docker.service" >> "$DEST/$version/Dockerfile.build"
```

Comment:
will this lead to `-unsupported` being added to the version string? https://github.com/docker/docker/blob/cf91a1be45fc4f1d4e3338610f921d665d406b04/hack/make.sh#L84-L94